### PR TITLE
use less goroutine and memory while executing multiple cmds at once

### PR DESCRIPTION
**Pull request by Jaeyo**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

too many goroutine while executing multiple cmds at once

### CODE REVIEW

**Comment by Jaeyo** [comment]
@danielnelson can you review again whether your requesting change all adopted?

File: internal/internal.go [hunk 1]
```diff
@@ -153,22 +153,26 @@ func RunTimeout(c *exec.Cmd, timeout time.Duration) error {
...
-	timer := time.NewTimer(timeout)
-	done := make(chan error)
-	go func() { done <- c.Wait() }()
-	select {
-	case err := <-done:
-		timer.Stop()
-		return err
-	case <-timer.C:
-		if err := c.Process.Kill(); err != nil {
+	var err error
```
Comment (danielnelson) [inline internal/internal.go]: This error is never assigned to, remove it and the logic checking it below.

File: internal/internal.go [hunk 2]
```diff
@@ -153,22 +153,26 @@ func RunTimeout(c *exec.Cmd, timeout time.Duration) error {
...
+		if err != nil {
 			log.Printf("E! FATAL error killing process: %s", err)
-			return err
+			return
 		}
-		// wait for the command to return after killing it
-		<-done
+	})
+
+	exitErr := c.Wait()
```
Comment (danielnelson) [inline internal/internal.go]: If you remove the err above, this is the only error so I would probably just call it `err`.

**Comment by Jaeyo** [comment]
@danielnelson Done! :D

**Pull request merged**